remote_templates/commit0.example.yml

```diff
@@ -0,0 +1,24 @@
+name: hello-world
+
+# Context will populated automatically or could be added manually
+context: 
+  awsCognito: 
+    poolID: 123
+    clientID: ABC
+
```

Comment:
Note that the order of the package matter. Certain packages might require context generated from pervious packages. Eg. typically depending on infrastructure terraform outputs.